reconcile/utils/gql.py

```diff
@@ -91,20 +83,13 @@ def __init__(self, url, token=None, int_name=None, validate_schemas=False):
             if not self._valid_schemas:
                 raise GqlApiIntegrationNotFound(int_name)
 
-    @retry(exceptions=GqlApiError, max_attempts=5)
```

Comment:
Removing the `@retry` decorator. I feel we should let it crash every time the server is not reachable in order not to obscure the errors.

Reply:
@maorfr do you have any reasons for not doing this?

Reply:
network issues?

if we're looking for server issues, let's instrument the server IMO

Reply:
would you be aginst trying this out? I'd like to. We can add the decorator if we see a lot of errors.